core/src/main/java/dk/alexandra/fresco/logging/PerformanceLogger.java

```diff
@@ -17,19 +19,44 @@
   public DecimalFormat df = new DecimalFormat("#.00");
   
   public enum Flag {
-    LOG_NETWORK, LOG_RUNTIME, LOG_NATIVE_BATCH;
+    LOG_NETWORK, LOG_EVALUATOR, LOG_NATIVE_BATCH;
 
     public static final EnumSet<Flag> ALL_OPTS = EnumSet.allOf(Flag.class);
   }
 
   /**
-   * Prints any performance numbers picked up.
+   * Prints any performance numbers picked up using the
+   * default logger.
    */
-  public abstract void printPerformanceLog(int myId);
+  public default void printPerformanceLog(int myId) {
+    printToLog(PerformanceLogger.log, myId);
+  }
+  
+  /**
+   * Prints any performance numbers picked up using the
+   * specified logger. 
+   */
+  public void printToLog(Logger log, int myId);
```

Comment:
I really dont like this construction. It complicates the interface and is only there for testing purposes.